data-prepper-plugins/decompress-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/decompress/DecompressProcessorConfig.java

```diff
@@ -31,6 +32,9 @@ public class DecompressProcessorConfig {
     @JsonPropertyDescription("The type of decompression to use for the keys in the event. Only <code>gzip</code> is supported.")
     @JsonProperty("type")
     @NotNull
+    @ExampleValues({
```

Comment:
Let's avoid example values for enumerations. The enumeration set is the full list of examples.